client/src/index.ejs

```diff
@@ -7,7 +7,9 @@
     <link rel="shortcut icon" href="/favicon.ico">
     <script src="https://use.fontawesome.com/51ad00bb8c.js"></script>
     <script async data-cfasync='false' src='https://d29l98y0pmei9d.cloudfront.net/js/widget.min.js?k=Y2xpZW50SWQ9NzQ3Jmhvc3ROYW1lPWFrdm9sdW1lbi5zdXBwb3J0aGVyby5pbw=='></script>
-    <script async defer data-bugyard='5e65daa3f74c520014b05cfa' src='https://widget.bugyard.io/bugyard.min.js'></script>
+    <script type="text/javascript">
+      !function(){if("function"!=typeof window.bugyard){var a=function(){a.c(arguments)};a.q=[],a.c=function(b){a.q.push(b)},window.bugyard=a;var b=document.createElement("script");b.setAttribute("data-bugyard","5e65daa3f74c520014b05cfa"),b.setAttribute("async","async"),b.setAttribute("defer","defer"),b.setAttribute("src","https://widget.bugyard.io/bugyard.min.js"),document.getElementsByTagName("head")[0].appendChild(b)}}();
```

Comment:
It would be interesting to understand why the change? Why do we need a script to include an external `.js`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script
````js
!(function() {
  if ("function" != typeof window.bugyard) {
    var a = function() {
      a.c(arguments);
    };
    (a.q = []),
      (a.c = function(b) {
        a.q.push(b);
      }),
      (window.bugyard = a);
    var b = document.createElement("script");
    b.setAttribute("data-bugyard", "5e65daa3f74c520014b05cfa"),
      b.setAttribute("async", "async"),
      b.setAttribute("defer", "defer"),
      b.setAttribute("src", "https://widget.bugyard.io/bugyard.min.js"),
      document.getElementsByTagName("head")[0].appendChild(b);
  }
})();

````

Reply:
I thinkk that using HTML5 `async` and `defer` for the script tag has the same effect as injecting it via a `document.createElement("script")` as we're not including the same script more than once -  https://stackoverflow.com/a/14666847